.github/workflows/ci.yml

```diff
@@ -189,3 +189,40 @@ jobs:
         run: exit 0
       - if: ${{  (contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')) }}
         run: exit 1
+
+  antithesis:
+    # if: github.ref == 'refs/heads/owi/main'
+    runs-on: ubuntu-latest
+    environment: antithesis
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2147483647
+      - name: Install dependencies
+        run: sudo apt install -y jq curl
+      - uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: "21"
+      - name: Login to GitHub Container Registry
+        run: docker login --username tigerbeetle --password "${{ secrets.GITHUB_TOKEN }}" ghcr.io
+      - name: Download Zig
+        run: ./zig/download.sh
+      - name: Build and push
+        run: ./zig/zig build scripts -- antithesis --tag='antithesis-latest' --push
+      # - run: |
+      #     body=$(jq -cn \
+      #       --arg images "${images}" \
+      #       --arg config_image "${config_image}" \
+      #       --arg git_commit "${git_commit}" \
+      #       '{params: {"custom.duration":"0.5", "antithesis.images": $images, "antithesis.config_image": $config_image, "antithesis.description": $git_commit, "antithesis.report.recipients":"oskar@tigerbeetle.com"}}')
+      #
+      #     curl --fail \
+      #       -u "$ANTITHESIS_USER:ANTITHESIS_PASS" \
+      #       -X POST https://tigerbeetle.antithesis.com/api/v1/launch_experiment/tigerbeetle \
+      #       -d "$body"
+      #   env:
+      #     ANTITHESIS_USER: ${{ secrets.ANTITHESIS_USER }}
+      #     ANTITHESIS_PASS: ${{ secrets.ANTITHESIS_PASS }}
+      #
```

Comment:
Up to you, but consider, instead of introducing a separate CI job here, to chain-load this onto the existing `devhub` one. 

The principle here is 

https://matklad.github.io/2023/12/31/O(1)-build-file.html

Rather than adding new build hooks, add stuff to an existing hook. For antithesis, the proper hook seems to be "new code is pushed to `main`", which is our `devhub` job (maybe its worth to rename `devhub` job to "run on main" or something)